fluss-lake/fluss-lake-paimon/src/main/java/com/alibaba/fluss/lake/paimon/tiering/PaimonLakeCommitter.java

```diff
@@ -74,6 +71,14 @@ public PaimonCommittable toCommittable(List<PaimonWriteResult> paimonWriteResult
         for (PaimonWriteResult paimonWriteResult : paimonWriteResults) {
             committable.addFileCommittable(paimonWriteResult.commitMessage());
         }
+        if (!paimonWriteResults.isEmpty()) {
+            committable.addProperty(
```

Comment:
This only add the bucket offsets for the records written in this snapshot delta. But we definetly need the full bucket offsets since when snapshot expire, we'll lose the previous bucket offsets .